src/ansys/speos/core/source.py

```diff
@@ -884,44 +1232,65 @@ def __init__(
             if default_values:
                 # Default values
                 self._exitance_variable.SetInParent()
-                self.set_axis_plane()
+                self.axis_plane = [0, 0, 0, 1, 0, 0, 0, 1, 0]
```

Comment:
i hope thats correct...
```suggestion
                self.axis_plane = ORIGIN[0:8]
```